workshop.md

```diff
@@ -260,20 +275,39 @@ Now that we have the Markdown files in our Lakehouse, we can use Azure OpenAI to
 
 </div>
 
-On your notebook, add the following code to configure the Azure OpenAI client:
+On your notebook, add the following code to install the openai library, run the following command:
+
+```python
+%pip install openai
```

Comment:
This code should be moved to the top when we install the QR Codes library.

Reply:
fixed this by moving it to the  install libraries section